javascript-data-model/main.js

```diff
@@ -0,0 +1,63 @@
+/* exported orderHistory */
+
+var orderHistory = [
+  {
+    orderInfo: {
+      booksOrdered: {
```

Comment:
Instead of differentiating between `booksOrdered` and `gameOrders`, I think this can be simplified by just having a list of all items ordered in each order 🤔